tools/crowdin/actions/pr-creator/index.js

```diff
@@ -0,0 +1,85 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+(async () => {
+  try {
+    const branch = core.getInput('branch-name');
+    const [owner, repo] = core.getInput('owner-repo').split('/');
+    if (!owner || !repo) {
+      core.setFailed('Must specify a valid ownerName/repoName');
+    }
+    const base = core.getInput('base');
+    const title = core.getInput('title');
+    const body = core.getInput('body');
+    const labelsStr = core.getInput('labels');
+    const labels = labelsStr.trim().split(/,\s+/);
+    const reviewersStr = core.getInput('reviewers');
+    const reviewers = reviewersStr.trim().split(/,\s+/);
```

Comment:
```suggestion
    const reviewers = reviewersStr.trim().split(/,\s+/);
    
    const github = githubRoot.getOctokit(token);
```

So the `github` context in the `actions/github-script` I was using is assigned to an Octokit client. Digging through the docs, this is how to get that same client.

Assigned it to `github` so we don't have to modify all the names later in the code.